### ISSUE TITLE
disable-(next-)-line does not work for long comment lines

### ISSUE THREAD

**Issue by olivierperes** [mgechev/revive#98]
**Describe the bug**
`revive:disable-line` and `revive:disable-next-line` do not work with rule `line-length-limit` when applied to comments, although they work fine with lines of code.

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive`
2. I run it with the following flags & configuration file:

```shell
# flags

revive -config myconfig.toml
```

```toml
# config file

[rule.line-length-limit]
  arguments = [80]
```

```go
package main

import "fmt"

// revive:disable-next-line:line-length-limit
// The length of this comment line is over 80 characters, this is bad for readability.

func main() {
	// revive:disable-next-line:line-length-limit
	fmt.Println("This line is way too long. In my opinion, it should be shortened.")
}
```

**Expected behavior**
`revive` should output nothing, since the line length rule was disable for both offending lines.

**Desktop (please complete the following information):**
 - OS: Mint 19.1 (based on Ubuntu 18.04)
 - Version of Go: 1.11.4

**Comment by chavacava** [comment]
Hi @olivierperes, I can reproduce the problem. I'll work on a fix.
Thanks!
